charts/buffer/values.yaml

```diff
@@ -98,3 +99,15 @@ proxy:
     repository: broadinstitute/openidc-proxy
     # proxy.image.version -- Proxy image tag
     version: bernick_tcell
+  tcell:
+    # proxy.tcell.enabled -- Enables TCell
+    enabled: true
+    # proxy.tcell.vaultPrefix -- Prefix for TCell secrets in vault. Required if proxy.tcell.enabled is true.
+    vaultPrefix:
+    # proxy.tcell.hostIdentifier -- Identifier used for logging to TCell. Required if proxy.tcell.enabled is true
+    hostIdentifier:
```

Comment:
I think you need a host identifier of some sort. name of the service and it's env is good. "buffer-prod" or somethign.

Reply:
That value gets set from the [terra-helmfile](https://github.com/broadinstitute/terra-helmfile) repo, based on which environment the chart is being deployed to. For example, [this is where it gets populated for my env](https://github.com/broadinstitute/terra-helmfile/blob/61448a1f9fb94c6fe755e7da46f6a9d8f8966cdd/terra/values/buffer/personal/gmalkov.yaml#L14).